pkg/upstream/cluster/loadbalancer_test.go

```diff
@@ -0,0 +1,35 @@
+package cluster
```

Comment:
could you add a test case of benchmark ？

Reply:
done.